strings/paperwork.json

```diff
@@ -0,0 +1,199 @@
+		"subject_one is prototyping an engine that replaces the Supermatter.",
+		"subject_one is method that makes the Hypertorus Fusion Reactor safer.",
+		"subject_one is illegally working on modernizing cloning technology.",
+		"subject_one is creating firing pens that makes weapons less effective.",
+		"subject_one is creating firing pens that makes weapons more effective.",
+		"subject_one is working with victim to create technology that rivals subject_two's inventions.",
+		"subject_one is developing toys aimed for children.",
+		"subject_one is developing hardlight weaponry that aims to replace the Bluespace Artillery.",
+		"subject_one is patenting a new technology that replaces current AI technology.",
+		"subject_one is patenting a new technology that enhances current AI technology.",
+		"subject_one is developing new bluespace radio technology that enhances telecommunications.",
+		"subject_one has begun development on technology that prevents mothpeople from walking into the Supermatter chamber.",
+		"subject_one has begun development on tablecloths that prevents felinids jumping onto tables.",
+		"subject_one is working on a breakthrough of plasma research.",
+		"subject_one is patenting a new technology that enhances surgeries.",
+		"subject_one is patenting a new technology that makes bombs more explosive.",
+		"subject_one is patenting a new technology that makes less more explosive.",
+		"subject_one is patenting a new technology that makes plasma more viable.",
+		"subject_one is patenting a new technology that plasma less viable.",
+		"victim is creating a device for subject_one that will replace security officers.",
+		"victim is creating a device for subject_one that will replace botanists.",
+		"victim is creating a brand new fax machine for the galaxy's paper-pushers.",
+		"victim was removed from subject_one forcibly and is now working on their new patent at subject_two.",
+		"victim stole patents from subject_one forcibly and is sharing them with subject_two.",
+		"subject_one working on engines to make evacuation shuttles faster.",
+		"subject_one working on machines to make evacuation shuttles calmer.",
+		"subject_one has patented a revolutionary [REDACTED]."
+	],
+
+	"contents_random": [
+		"Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+		"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed bibendum venenatis arcu ac semper.",
+		"Proin commodo nibh purus, ut rhoncus nibh efficitur quis. Quisque euismod.",
+		"Nulla maximus cursus massa, vel bibendum mi sollicitudin eu. Proin at nisi vel mi vehicula.",
+		"Maecenas ullamcorper tempus dui, vel blandit urna viverra nec. Class aptent taciti sociosqu ad litora.",
+		"Leclerc's lawsuit.",
+		"Gah-Jeeliu's medical bill.",
+		"Rasha-Tanaka's requests form.",
+		"Joey Petunia's bombing charges.",
+		"Dendy Doom's missing degree.",
+		"Aza-Zure's missing report.",
+		"Mortimer v. Oxygen case report.",
```

Comment:
Also to add to Melberts comment, these lines specifically 169 ~ 180 (will they show properly after I post this?)